packages/app/public/config.js

```diff
@@ -5,4 +5,5 @@ window.KEYCLOAK_REDIRECT_URI = 'http://localhost:3000/keycloak/callback';
 window.GRAPHQL_URI = 'http://localhost:8090/graphql';
 window.REST_URI = 'http://localhost:8090/api';
 window.SHOW_INHABITANT_AMOUNT = 'true';
-window.ADDRESS_RESEARCH_URL = 'https://www.google.nl';
+window.MENU_POLLING_INTERVAL = '10000';
```

Comment:
Its more like "messages" polling right?

Reply:
in the scope of messages, yes. but overall, since the polling happens in the menuitem, it can be applied to any menuitem. Let's say we want to show a  ❗️next to My Cases when there's an update, same polling var can be used across the whole menu since I don't see a reason for it to poll faster or slower than any other menu item.

Reply:
Would you prefer this to be called `MESSAGES_COUNT_POLLING_INTERVAL` ?